mesa/entities.py

```diff
@@ -11,6 +11,17 @@ def __init__(self, size=0, shape='null'):
         self.size: int = size
 
 
+class Topic:
+
+    def __init__(self,  name="Topic", risk=0.0, protective=0.0):
```

Comment:
This default value assignment is only for development right? If so let's write it down.

Reply:
Ho messo i veri valori di default